electron/console.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Use this module to log things into the console.
+ * Everything will be saved to a file before the app exits.
+ * Note that with console.log and console.warn everything will be saved too.
+ * error equals console.error
+ */
+
+
```

Comment:
Can you wrap this into a single function? Thinking of
```
type LogLevel = "error" | "warning" | "info" | "debug";
export function logMessage(level: LogLeveL, message: string) {
  switch(level){
    case 'error':
      console.log('ERROR: ' + message);
      break;
// ...
}
```

Also some documentation with jsdoc or tsdoc would be great :).

Reply:
I think that the current solution is more idiomatic. Think of window.console or electron-log. They do the same.

Reply:
There are docs already :)

If you think that docs are missing, where? Thank you.

Reply:
I've just added documentation. If you think that docs are missing yet, tell me where! Thank you.